platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/service/PlatformStateFacade.java

```diff
@@ -31,51 +31,49 @@
  */
 public class PlatformStateFacade {
```

Comment:
If this becomes a utility class, I would make it final and add a private constructor to prevent initialization.
Optional: rename to `PlatformStateUtils`

```suggestion
public final class PlatformStateFacade {

    private PlatformStateFacade() {}
```